openid-risc-profile-specification-1_0.txt

```diff
@@ -53,9 +53,9 @@ Table of Contents
 
 
 
-Scurtescu, et al.         Expires May 16, 2022                  [Page 1]
+Scurtescu, et al.        Expires August 8, 2022                 [Page 1]
```

Comment:
Can you explain how this date is chosen?

Reply:
+6m